test/__snapshots__/test.js.snap

```diff
@@ -1,5 +1,75 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`with processors AsyncCallbackInterface.webidl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+const interfaceName = \\"AsyncCallbackInterface\\";
+
+exports.convert = function convert(value, { context = \\"The provided value\\" } = {}) {
+  if (!utils.isObject(value)) {
+    throw new TypeError(\`\${context} is not an object.\`);
+  }
+
+  const { wrapper } = {
+    wrapper(...args) {
+      let completion;
+      let thisArg = this;
+      let O = value;
+      let X = O;
+
+      try {
+        if (typeof O !== \\"function\\") {
+          X = O[\\"asyncMethod\\"];
+          if (typeof X !== \\"function\\") {
+            throw new TypeError(\`\${context} does not correctly implement AsyncCallbackInterface.\`);
+          }
+          thisArg = O;
+        }
+
+        for (let i = 0; i < args.length; i++) {
+          args[i] = utils.tryWrapperForImpl(args[i]);
+        }
+        if (args.length < 0) {
+          for (let i = args.length; i < 0; i++) {
+            args[i] = undefined;
+          }
+        } else if (args.length > 0) {
+          args.length = 0;
+        }
+
+        let callResult = Reflect.apply(X, thisArg, args);
+
+        callResult = Promise.resolve(callResult).then(
+          value => {
+            value = conversions[\\"any\\"](value, { context: context + \\" promise value\\" });
+
+            return value;
+          },
+          reason => reason
+        );
+
+        return callResult;
+      } catch (err) {
+        return Promise.reject(err);
+      }
+    }
+  };
+
+  Object.defineProperties(wrapper, {
+    [utils.wrapperSymbol]: { value },
+    value: { value, enumerable: true }
+  });
+
+  return wrapper;
```

Comment:
Is wrapper the right name? Aren't we converting the incoming JS value into an internal representation for jsdom to use, not a jsdom-user exposed wrapper?

Reply:
I used the same name as @TimothyGu in <https://github.com/jsdom/webidl2js/pull/123>

Reply:
Hmm, well, it seems like the wrong name, so since this is closer to landing, let's change it. "converted" would be OK, or "webIDLRepresentation" maybe, or something like that...

Reply:
I went with `internalWrapper` before seeing this.